plugins/woocommerce/client/blocks/tests/e2e/tests/add-to-cart-with-options/add-to-cart-with-options.block_theme.spec.ts

```diff
@@ -162,6 +162,41 @@ test.describe( 'Add to Cart + Options Block', () => {
 		} );
 	} );
 
+	test( 'allows adding grouped products to cart', async ( {
+		page,
+		pageObject,
+		editor,
+	} ) => {
+		await pageObject.setFeatureFlags();
+
+		await pageObject.updateSingleProductTemplate();
+
+		await editor.saveSiteEditorEntities( {
+			isOnlyCurrentEntityDirty: true,
+		} );
+
+		await page.goto( '/logo-collection' );
+
+		const increaseQuantityButton = page.getByLabel(
+			'Increase quantity of Beanie'
+		);
+		await increaseQuantityButton.click();
+
+		const addToCartButton = page.getByText( 'Add to cart' ).first();
+
+		await addToCartButton.click();
+
+		await page.waitForResponse(
+			( response ) =>
+				response.url().includes( 'wp-json/wc/store/v1/cart' ) &&
+				response.status() === 200
+		);
+
+		await expect( page.getByText( 'Added to cart' ).first() ).toBeVisible();
```

Comment:
Is `.first()` needed here? I think there is only one `Added to cart` on the page, no? :thinking:

Reply:
Yes that's right, thank you! Removed in https://github.com/woocommerce/woocommerce/pull/58727/commits/801c22ea20e9763218307bd1cb0c8bfefd8bb0f9.